### DISCUSSION TITLE
你们的声音, 我们听到了! 蓝灯团队正在努力解决问题! 💪

### DISCUSSION THREAD

**Discussion by MaxMKMax** [getlantern/lantern-forum-cn#40] [Announcements]
![image (1)](https://github.com/getlantern/lantern-forum-cn/assets/155475142/6d1b4bd0-55cb-4731-b203-336947cda8ad)

**Comment by hacker873** [comment]
今天发现更新了7.7.2的版本，勉强能用，服务器跳到DE了，速度很慢，希望能尽快解决问题，对大陆客户来说SG的服务器是最好用的

**Reply by MaxMKMax** [reply]
@hacker873 好的。完全明白这个问题的存在。我理解你们都想要SG服务器。

我们已经正在和科技部反馈了。给我们一点时间。尽快弄好。非常抱歉。

**Comment by Frank859** [comment]
Windows版本7.7.2(20240118.145451)一直是连接中，而且不断在尝试下载proxies.yaml.gz这个文件但一直不成功，下载地址是http://config.getiantem.org/proxies.yaml.gz，感觉这个地址是不是有问题？

**Comment by rochesterfd** [comment]
请问管理员，为何我已更新到7.7.2，但依然连接的是HK的server？

**Comment by MaxMKMax** [comment]
@hacker873 @rochesterfd @Frank859 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

在这之前，您方便让我知道您现在用的是哪个版本吗？打个比方version xxx.xxx . 然后用安卓手机还是萍果手机? 或者哪款的电脑呢？

然后, 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
